goto/commands/open.py

```diff
@@ -28,4 +28,4 @@ def open(magic, args):
     except subprocess.CalledProcessError as e:
         return None, GotoError("open_failed", message=e.message)
 
-    return None, None
+    return "Opened folder: %s" % url, None
```

Comment:
open can also open files, not only folders. 

Try it with a textfile or a pdf for instance ^^

Reply:
Yes, or gif, or image

How about?
```suggestion
    return "Opening: %s" % url, None
```

Reply:
Perfecto!